From 27ba838b5e67ef9d8a27690b55d0c37113734172 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Wed, 7 Feb 2018 18:51:10 +0800
Subject: [PATCH] power/rk817_charger: fix compile warning

drivers/power/rk817_charger.c:735:2: warning: this else clause
does not guard... [-Wmisleading-indentation]

Change-Id: Ib65d2f54463c39d7fe9fd1dfb39a76035df05172
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 drivers/power/rk817_charger.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/power/rk817_charger.c b/drivers/power/rk817_charger.c
index 01fda7531a4c..c2c426e608f0 100644
--- a/drivers/power/rk817_charger.c
+++ b/drivers/power/rk817_charger.c
@@ -736,7 +736,7 @@ static void rk817_charge_set_chrg_finish_condition(struct rk817_charger *charge)
 		rk817_charge_set_term_current_digital(charge,
 						      charge->chrg_finish_cur);
 
-		rk817_charge_set_chrg_term_mod(charge, charge->chrg_term_mode);
+	rk817_charge_set_chrg_term_mod(charge, charge->chrg_term_mode);
 }
 
 static int rk817_charge_online(struct rk817_charger *charge)
-- 
2.35.3

